ocs_ci/ocs/cnv/virtual_machine.py

```diff
@@ -93,6 +94,7 @@ def create_vm_workload(
         access_mode=constants.ACCESS_MODE_RWX,
         pvc_size="30Gi",
         source_url=constants.CNV_CENTOS_SOURCE,
+        pvc_obj=None,
```

Comment:
Parikshit also added same parameter in his PR. It will conflict. https://github.com/red-hat-storage/ocs-ci/pull/11045